app/controllers/organisations_controller.rb

```diff
@@ -2,7 +2,7 @@ class OrganisationsController < ApplicationController
   before_action :index, :authorise_admin
 
   def index
-    @all_organisations = Organisation.all
+    @all_organisations = Organisation.order('name ASC').all
```

Comment:
I would write this as `.order(:name)`, iirc the other syntax can cause issues.

I'm happy to lose the explicit callout that it's `ASC` order , as it's rails convention that `.order` will produce an alphabetical list in this case.